components/batch-system/src/metrics.rs

```diff
@@ -11,6 +40,55 @@ lazy_static! {
     )
     .unwrap();
 
+    pub static ref FSM_RESCHEDULE_COUNTER_VEC: IntCounterVec = register_int_counter_vec!(
```

Comment:
Why aren't these reschedule metrics shown in the graph of your comments?

Reply:
the screen is not big enough :)